### PR TITLE
Fix deprecated methods

### PR DESCRIPTION
**Pull request by qmonmert**
https://javadoc.io/doc/io.jsonwebtoken/jjwt-api/latest/deprecated-list.html

### CODE REVIEW

File: src/main/resources/generator/server/springboot/mvc/security/jwt/authentication/test/infrastructure/primary/JwtReaderTest.java.mustache [hunk 1]
```diff
@@ -87,7 +86,7 @@ class JwtReaderTest {
     return Jwts
       .builder()
       .subject("test")
-      .signWith(KEY, SignatureAlgorithm.HS512)
+      .signWith(KEY, Jwts.SIG.HS256)
```
Comment (murdos) [inline src/main/resources/generator/server/springboot/mvc/security/jwt/authentication/test/infrastructure/primary/JwtReaderTest.java.mustache:89]: Is there a reason for switching from HS512 to HS256?

Comment (qmonmert) [reply src/main/resources/generator/server/springboot/mvc/security/jwt/authentication/test/infrastructure/primary/JwtReaderTest.java.mustache:89]: yes it's deprecated 
image

Comment (murdos) [reply src/main/resources/generator/server/springboot/mvc/security/jwt/authentication/test/infrastructure/primary/JwtReaderTest.java.mustache:89]: SIG.HS512 is not deprecated: https://javadoc.io/doc/io.jsonwebtoken/jjwt-api/latest/io/jsonwebtoken/Jwts.SIG.html

Comment (qmonmert) [reply src/main/resources/generator/server/springboot/mvc/security/jwt/authentication/test/infrastructure/primary/JwtReaderTest.java.mustache:89]: You don’t understand, it’s the signature which is deprecated like I said in the commit. Generate an app and you will see

Comment (qmonmert) [reply src/main/resources/generator/server/springboot/mvc/security/jwt/authentication/test/infrastructure/primary/JwtReaderTest.java.mustache:89]: image

image

Comment (murdos) [reply src/main/resources/generator/server/springboot/mvc/security/jwt/authentication/test/infrastructure/primary/JwtReaderTest.java.mustache:89]: You should really read what I wrote, twice.
It's not about the signature change, it's about the change of the length of the key, without justification (your commit doesn't explain why you're changing the length of the key).
```suggestion
      .signWith(KEY, Jwts.SIG.HS512)
```
(and yes, I've generated the app, I've seen that using 512 bit still works)

**Pull request merged**